database/schema.sql

```diff
@@ -12,4 +12,24 @@ CREATE TABLE movies(
     sources_ids INT,
     PRIMARY KEY(id),
     CONSTRAINT fk_sources FOREIGN KEY(sources_ids) REFERENCES sources(id)
+);
+
+CREATE TABLE books(
+    id INT GENERATED ALWAYS AS IDENTITY,
+    publish_date DATE NOT NULL,
+    archived BOOLEAN,
+    cover_state VARCHAR(50),
+    publisher VARCHAR(80),
+    book_color VARCHAR(30),
+    labels_id INT,
+    PRIMARY KEY(id),
+    CONSTRAINT fk_labels FOREIGN KEY(labels_id) REFERENCES labels(id)
+);
+CREATE INDEX book_label_idx ON books(labels_id);
```

Comment:
- [ ] Kindly add the column `title` so the table created matches the structure of the Book class you created.

- [ ] Kindly verify the book_color column, it may not belong to the Book class.